index.html

```diff
@@ -0,0 +1,127 @@
+                    </li>
+                    <li class="b"><i class="C"><img src="img/2.png" alt=""></i>
+                        <p class="A">Lectures</p>
+                        <p class="B">Learn mordern techniques
+                            in our daily classes from monday to friday
+                            each week
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/3.png" alt=""></i>
+                        <p class="A">Programs</p>
+                        <p class="B">Enrol and become eligible for our
+                            government sponsored susidies
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/4.png" alt=""></i>
+                        <p class="A">Assistance</p>
+                        <p class="B">Access technical support
+                            form our technical teams
+                            available to you 24 hours a day
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/5.png" alt=""></i>
+                        <p class="A">Partners</p>
+                        <p class="B">Become acredited with us and gain
+                            the advantages that come with being an
+                            affiliate.
+                        </p>
+                    </li>
+                
+                </ul>
+                <button class="join" type="button">Attend The Summit</button>
+                <p class="psee" id="psee">SEE THE WHOLE PROGRAM</p>
+     </section>
+     <section class="speaker-sec">
+        <h2 class="speaker-header">Featured Guests</h2>
+        <div class="underline"></div>
+                <ul class="speaker-list" id="speaker-list">
+                   
+                </ul>
+            <button class="more-btn" id="more-btn" type="button">MORE<i class="fa fa-angle-down arrow"></i></button>
+    </section>
+    <section class="partner" id="partner">
+        <h2 class="partner-header">Partners</h2>
+        <div class="underline"></div>
+        <ul class="bottom-logos">
+            <li><a href="https://www.facebook.com/"><img src="img/faceb.png" alt="logo"></a></li>
+            <li><a href=""><img src="img/YouTube-Logo.png" alt="logo"></a></li>
+            <li><a href=""><img src="img/microverse.png" alt="logo"></a></li>
+            <li><a href=""><img src="img/paypal.png" alt="logo"></a></li>
+            <li><a href="https://www.skype.com/en/"><img src="img/alibab.png" alt="logo"></a></li>
+        </ul>
+            
+    </section>
+    <footer class="footer bg-color" id="footer">
+        <img src="img/iot.gif" alt="logo">
+        <div class="footer-text">
+            <span class="footer-copyright white-text">2022 @Iot Tech conference </span>
+            <span class="footer-about white-text">Technology Giants.</span>
+        </div>
+    </footer>
```

Comment:
- [ ] Good job on making the footer contain a logo aligned to the left and text aligned to the right,  but the spacings should match the original design.
- [ ] The background color should match the original design color. AT the moment, yours is dark while the original design is white. Please kindly adjust.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169510054-139c671f-1eaa-4ec1-84c3-c7743a14c348.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/169510180-84a5a9b5-43a3-4538-a80e-97d599a73e6d.png)